src/components/ChangePlanDeprecated/Checkout/PaymentMethod/PaymentMethod.js

```diff
@@ -70,14 +71,16 @@ const PaymentMethodField = ({ billingCountry, paymentMethodType, optionView, han
   const _ = (id, values) => intl.formatMessage({ id: id }, values);
 
   const allowTransfer = countriesAvailableTransfer.filter((c) => c === billingCountry)[0];
+  const allowMercadoPago = countriesAvailableMercadoPago.filter((c) => c === billingCountry)[0];
```

Comment:
why don't you use find instead of filter?

Reply:
good, I will replace to use find instead of filter

Reply:
Done